### PR TITLE
wires resource sets instead of resource router

### PR DESCRIPTION
**Pull request by xh3b4sd**
Towards https://github.com/giantswarm/giantswarm/issues/3664.

### CODE REVIEW

File: service/controller/aws/cluster.go [hunk 1]
```diff
@@ -206,23 +209,7 @@ func NewCluster(config ClusterConfig) (*Cluster, error) {
...
-	var clusterController *controller.Controller
-	{
-		c := controller.Config{
-			CRD:            v1alpha1.NewAWSClusterConfigCRD(),
-			CRDClient:      crdClient,
-			Informer:       newInformer,
-			Logger:         config.Logger,
-			ResourceRouter: resourceRouter,
-			RESTClient:     config.G8sClient.ProviderV1alpha1().RESTClient(),
+			RESTClient: config.G8sClient.ProviderV1alpha1().RESTClient(),
```
Comment (tuommaki) [inline service/controller/aws/cluster.go]: While changing this, could you maybe fix the client to use correct rest client? This should be:
`config.G8sClient.CoreV1alpha1().RESTClient()`. I know it was like that and it doesn't matter as both have decoders & stuff under the same configuration, but for the consistency still.

File: service/controller/azure/cluster.go [hunk 2]
```diff
@@ -200,23 +203,7 @@ func NewCluster(config ClusterConfig) (*Cluster, error) {
...
-	var clusterController *controller.Controller
-	{
-		c := controller.Config{
-			CRD:            v1alpha1.NewAzureClusterConfigCRD(),
-			CRDClient:      crdClient,
-			Informer:       newInformer,
-			Logger:         config.Logger,
-			ResourceRouter: resourceRouter,
-			RESTClient:     config.G8sClient.ProviderV1alpha1().RESTClient(),
+			RESTClient: config.G8sClient.ProviderV1alpha1().RESTClient(),
```
Comment (tuommaki) [inline service/controller/azure/cluster.go]: `CoreV1alpha` ...

File: service/controller/kvm/cluster.go [hunk 3]
```diff
@@ -201,23 +204,7 @@ func NewCluster(config ClusterConfig) (*Cluster, error) {
...
-	var clusterController *controller.Controller
-	{
-		c := controller.Config{
-			CRD:            v1alpha1.NewKVMClusterConfigCRD(),
-			CRDClient:      crdClient,
-			Informer:       newInformer,
-			Logger:         config.Logger,
-			ResourceRouter: resourceRouter,
-			RESTClient:     config.G8sClient.ProviderV1alpha1().RESTClient(),
+			RESTClient: config.G8sClient.ProviderV1alpha1().RESTClient(),
```
Comment (tuommaki) [inline service/controller/kvm/cluster.go]: `CoreV1alpha1` ...

File: service/controller/aws/cluster.go (hunk 1 above)
Comment (xh3b4sd) [reply service/controller/aws/cluster.go]: Addressed in https://github.com/giantswarm/cluster-operator/pull/198.

**Pull request merged**